dusk-3/sequencer/genesis.json

```diff
@@ -0,0 +1,81 @@
+{
+  "genesis_time": "2023-09-22T17:22:35.092832Z",
+  "chain_id": "astria-dusk-3",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "22020096",
+      "max_gas": "-1"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app": "0"
+    }
+  },
+  "validators": [
+    {
+      "address": "230592632006DB2733444BB6DE11DB3F4B2F9AE4",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "MkFfCdvuQpfMmoQcLCMSv5A/xTxIhg14iuZglzVaWF8="
+      },
+      "power": "1",
+      "name": "node0"
+    },
+    {
+      "address": "6F35496BCC8CF0EF9E2AC090FAEF578152549518",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "CeKTMbL61MvTZ5hoA0hKL1REQUhejnNhEtKtSbg2Vso="
+      },
+      "power": "1",
+      "name": "node1"
+    },
+    {
+      "address": "115F94D8C98FFD73FE65182611140F0EDC7C3C94",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "lvQ6hEiSjx5YCGTWn+ROCTxagqHUqAxZCG1+Z5ds2kU="
+      },
+      "power": "1",
+      "name": "node2"
+    },
+    {
+      "address": "60B5F4FED5CDE1F43E71CD03A4ACDCABB5724455",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "ixbsw2r9ME9ikcWTwWSdtk+AxNNQ4PyzOjNnyiMMe58="
+      },
+      "power": "1",
+      "name": "node3"
+    }
+  ],
+  "app_hash": "",
+  "app_state": {
+    "accounts": [
```

Comment:
These balances aren't the validators right, just the faucets and such? Is there any reason the validators would need to have a balance at genesis?

Reply:
Correct, the validators have no account balance. These are accounts used for faucets.